mobify/hora.js
# Dont track interactions by default

Status: **Ready for merge**
Owner: @ericmuyser
Reviewers: @scalvert 

## Todos:

### Feedback:
_none so far_

## How to Test
- Open page that sends events
- Check all GA events except 'pageview' are set as nonInteraction = 1.
